driver/modern_bpf/CMakeLists.txt

```diff
@@ -80,6 +76,8 @@ foreach(BPF_C_FILE ${BPF_C_FILES})
     get_filename_component(file_stem ${BPF_C_FILE} NAME_WE)
     set(BPF_O_FILE ${CMAKE_CURRENT_BINARY_DIR}/${file_stem}.bpf.o)
 
+## Please note: that the `libbpf` target exists only if we use `USE_BUNDLED_LIBBPF`
```

Comment:
There has to be a better way to do this, we don't need to do this in this PR but I'm pretty sure we can change the `find_library` statement in `libbpf.cmake` to give the same target we create when using the bundled variant.

Reply:
Oh yeah completely agree with you I'm not a cmake expert but i can try it hoping it wouldn't require too much time, in that case maybe i will move this fix in another PR, just to not block all the work for this

Reply:
I have not yet cleaned it, I have just added a TODO, I will do it in the next refactors if it's ok for you!

Reply:
Sounds good to me, no need to mix up build refactoring into a test PR 🙂